src/parser/wakefiles.cpp

```diff
@@ -199,8 +199,8 @@ static bool push_files_is_directory(int dirfd, const std::string &path, struct d
   return S_ISDIR(sbuf.st_mode);
 }
 
-#Recursively fills *out *with all paths under *dirfd *matching *re *.Automatically
-#skips directories that cannot have source files in them(.git, .build, .fuse).
+// Recursively fills *out *with all paths under *dirfd *matching *re *.Automatically
```

Comment:
What's with the asterisks in this and the above comments? They're confusing me.

Reply:
omg. They were right but the # "comment" plus the formatting moved them. 🤦 .